ethereum/blockcreation/src/main/java/org/hyperledger/besu/ethereum/blockcreation/MiningCoordinator.java

```diff
@@ -77,6 +81,26 @@ default boolean submitWork(final EthHashSolution solution) {
         "Current consensus mechanism prevents submission of work solutions.");
   }
 
+  /**
+   * Allows to submit the hashrate of a sealer with a specific id
+   *
+   * @param id of the sealer
+   * @param hashrate of the sealer
+   * @return true if the hashrate has been added otherwise false
+   */
+  default boolean submitHashRate(final String id, final Long hashrate) {
+    return false;
+  }
+
+  /**
+   * Returns a list of the current sealers
+   *
+   * @return a list of {@link SealerInfo}
+   */
+  default Map<String, SealerInfo> getSealerInfos() {
```

Comment:
I would remove this in favor or a new getHashRate() function that defaults to zero.

Reply:
Done